101-azure-portal-dashboard/azuredeploy.json

```diff
@@ -0,0 +1,268 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the existing virtual machine to show in the dashboard"
+      }
+    },
+    "virtualMachineResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the resource group that contains the virtual machine"
+      }
+    },
+    "dashboardName": {
+      "type": "string",
+      "defaultValue": "[newGuid()]",
```

Comment:
make this idempotent, e.g.

guid(parameters('virtualMachineName'), parameters('virtualMachineResourceGroup'))

Reply:
Thanks @bmoore-msft - have you reviewed the rest and this is the only change, or are you still reviewing?

Reply:
finished... the rest looks good

Reply:
@bmoore-msft - made the change, should be good to go